src/renderer/components/drawer/drawer-title.tsx

```diff
@@ -23,3 +23,14 @@ export class DrawerTitle extends React.Component<DrawerTitleProps> {
     );
   }
 }
+export class DrawerSubTitle extends React.Component<DrawerTitleProps> {
+  render() {
+    const { title, children, className } = this.props;
+
+    return (
+      <div className={cssNames("DrawerSubTitle", className)}>
+        {title || children}
+      </div>
+    );
+  }
+}
```

Comment:
Again, we shouldn't duplicate component above just for changing the class name. If we **really** need to use `DrawerSubTitle` we can use `DrawerTitle` and assign it some classname.

```
<DrawerTitle className="DrawerSubtitle"/>
```